microsoft-integration/pom.xml

```diff
@@ -16,31 +16,22 @@
 
 	<properties>
 		<!-- Dependency versions -->
-		<msal4j.version>1.11.3</msal4j.version>
-		<azure-identity.version>1.5.0</azure-identity.version>
-		<microsoft-graph.version>5.20.0</microsoft-graph.version>
+		<azure-identity.version>1.8.3</azure-identity.version>
```

Comment:
Shall we use [1.9.0](https://mvnrepository.com/artifact/com.azure/azure-identity/1.9.0)?